deploy/charts/litellm-helm/templates/deployment.yaml

```diff
@@ -102,7 +102,7 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
             - name: HOST
-              value: "::"
+              value: "{{ .Values.listen | default "::" }}"
```

Comment:
would this work for ipv4 addresses ?

Reply:
you just have to set the value to 0.0.0.0 for ipv4

Reply:
I think the default should be the original `0.0.0.0`, as this PR addresses a regression introduced when it was replaced with `::`